include/vkd3d_device_vkd3d_ext.idl

```diff
@@ -35,6 +35,17 @@ interface ID3D12DeviceExt : IUnknown
     HRESULT CaptureUAVInfo(D3D12_UAV_INFO *uav_info);
 }
 
+[
+    uuid(13228264-a243-42dd-b9e3-12dc8856d5f9),
+    object,
+    local,
+    pointer_default(unique)
+]
+interface ID3D12DeviceExt2 : ID3D12DeviceExt
```

Comment:
Nitpicking, but I believe Microsoft's naming convention for D3D interfaces goes `ID3D12Thing` → `ID3D12Thing1` → `ID3D12Thing2` → … so this should be `ID3D12DeviceExt1` instead if we are to stay true to it.

Reply:
Thanks for the catch. Addressed.